src/main/java/uk/gov/cshr/utils/ApplicationConstants.java

```diff
@@ -10,8 +10,13 @@
 
```

Comment:
Ensure the error messages are the same as what was agreed with CSL as part of the content review